### ISSUE TITLE
add users to usergroup

### ISSUE THREAD

**Issue by dauzeine** [chamilo/chamilo-lms#4839]
**Describe**
When I try to add a single user into a usergroup, the table at left is empty (it should contain the list of users not yet subscribed in the group class).

**To Reproduce**
1. Go to Administration / Class
2. Click on the first icon at right beside the name of a class
3. See error (blank table at right).

**Expected behavior**
The table at left should contain the list of users not yet subscribed.

**Screenshots**
![image](https://github.com/chamilo/chamilo-lms/assets/31802799/8c4c6653-9d1b-4c45-9b9a-2d74d406d007)

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu 22.04 LTS and Windows 11
 - Browser: chrome, firefox

**Server
 - OS: Linux Ubuntu 22.04 LTS, updated
 - Version of Chamilo: 1.11.20
 - Version of PHP: 7.4.3x

**Comment by ywarnier** [comment]
Sorry about that. Due to speed issues with very large portals, we have introduced an option in configuration.php that you have to enable to get the full list again (otherwise you have to search with at least 3 letters to get the corresponding names): 
```
// Show all users in selector as initial list when subscribing users to class
$_configuration['usergroup_add_user_show_all_student_by_default'] = true;
```

If you add the second line to your configuration.php, users will reappear. Thank you for reporting! We should add a term to explain this behaviour, but we're very late in the 1.11 development cycle, so it will probably be something to fix in 2.0.

**Comment by dauzeine** [comment]
Works like a charm, thanks Yannick (the 2 things: the search with the first 3 letters and the tweak of configuration.php). Sorry I missed it in configuration.dist.php.
